eng/targets/Helix.targets

```diff
@@ -11,7 +11,7 @@
   <PropertyGroup Condition="'$(TestDependsOnPlaywright)' == 'true'">
     <SkipHelixQueues>
       (Alpine.312.Amd64.Open)Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.12-helix-20200908125345-56c6673;
-      (Fedora.33.Amd64.Open)Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-33-helix-20210120000908-a9df267;
+      (Fedora.34.Amd64.Open)ubuntu.1604.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-34-helix-20210728124700-4f64125;
```

Comment:
Why are we using EOL Ubuntu 16.04 as a base for anything❔ That OS hasn't been supported since April.

Reply:
It's the only queue that supports msquic, @wfurt set it up - why use ubuntu16 instead of 18?